app/src/main/java/com/github/se/icebreakrr/ui/profile/ProfileEdit.kt

```diff
@@ -126,14 +139,16 @@ fun ProfileEditingScreen(
           Column(
               modifier = Modifier.padding(it).padding(padding).testTag("profileEditScreenContent"),
               horizontalAlignment = Alignment.CenterHorizontally) {
-                AsyncImage(
-                    model = profilePictureUrl,
-                    contentDescription = "Profile Picture",
-                    modifier =
-                        Modifier.size(profilePictureSize)
-                            .clip(CircleShape)
-                            .background(MaterialTheme.colorScheme.primary)
-                            .testTag("profilePicture"))
+                // A composable that allows the user to preview and edit a profile picture
+                ProfilePictureSelector(
+                    url = user.profilePictureUrl,
+                    size = profilePictureSize,
+                    onSelectionSuccess = { uri ->
+                      profilesViewModel.processAndUploadImage(context, uri)
+                    },
+                    onSelectionFailure = {
```

Comment:
This appears even when I cancel the picture selection (which is technically not a failure though)

Reply:
As I said in the Description of the PR, this was made on purpose to avoid problems. I may have found a way to make discarding work anyway for the profile picture, so I'll ask the rest of the team and change if the majority wants it.

Reply:
The more I think of it, the more I think it is right to discard the picture with the rest, even if it requires more work. I am currently working on a fix for that

Reply:
Ok, this is now fixed and clean, but now we have to solve glitches appearing in the `ProfileView` screen. I have some solutions to discuss during our next meeting (Monday)